components/infieldprogresscircle/project.json

```
@@ -0,0 +1,16 @@
+{
+	"$schema": "../../node_modules/nx/schemas/project-schema.json",
+	"name": "infieldprogresscircle",
+	"tags": ["component"],
+	"targets": {
+		"build": {},
+		"clean": {},
+		"compare": {},
+		"format": {},
+		"lint": {},
```

```suggestion
"lint": {},
"report": {},
```

Looking at some other project.jsons, I think we need a `"report": {}` line.